core/src/lib.rs

```diff
@@ -26,7 +26,10 @@ pub mod crypto {
     }
 
     pub mod random {
-        pub use crate::random::*;
+        pub use miden_crypto::rand::RpoRandomCoin;
+        pub use winter_crypto::{
+            DefaultRandomCoin as WinterRandomCoin, RandomCoin, RandomCoinError,
+        };
```

Comment:
Should we pull these through `miden-crypto` as well?

Reply:
Makes sense! I have re-exported the default random coin in a PR against the crypto crate.

Reply:
Thank you! https://github.com/0xPolygonMiden/crypto/pull/241 is now merged!